nwb_conversion_tools/schemas/yaml_specification_schema.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "yaml.specification.schema.json",
+  "title": "YAML specification schema",
```

Comment:
Can you make these more specific?

Reply:
I see how non-specific that is; changed all references to this as 'YAML conversion specification' to be clearer.